modmesh/core.py

```diff
@@ -74,11 +76,11 @@
     'Bezier3dFp64',
     'WorldFp32',
     'WorldFp64',
-
-    # TODO: have a way to enable this according to `USE_PYTEST_HELPER_BINDING`
-    'testhelper',
 ]
 
+if config.USE_PYTEST_HELPER_BINDING:
```

Comment:
load `testhelper` based on the CMake flag.

Reply:
Instead of using a `modmesh.config`, what if testing for the test module and loading accordingly?  If the test module is available, load it, if not, make a fake symbol.